Share The Load/Screens/ViewController.swift

```diff
@@ -0,0 +1,102 @@
+import RealmSwift
+
+class ViewController: UIViewController {
+    
+    private let getRealmQuestionsButton = UIButton()
+    private let removeFromRealmButton = UIButton()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        fetchQuiz()
+        createGetRealmQuestionsButton()
+        createRemoveButton()
+    }
+    
+    
+    private func createGetRealmQuestionsButton() {
+        view.addSubview(getRealmQuestionsButton)
+        getRealmQuestionsButton.translatesAutoresizingMaskIntoConstraints = false
+        getRealmQuestionsButton.setTitle("Get Questions From Realm", for: .normal)
+        getRealmQuestionsButton.frame = view.frame
+        getRealmQuestionsButton.backgroundColor = .systemBlue
+        getRealmQuestionsButton.layer.cornerRadius = 20
+        getRealmQuestionsButton.addTarget(self, action: #selector(buttonTapped), for: .touchUpInside)
+        
+        NSLayoutConstraint.activate([
+            getRealmQuestionsButton.centerYAnchor.constraint(equalTo: view.centerYAnchor),
+            getRealmQuestionsButton.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 20),
+            getRealmQuestionsButton.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -20),
+            getRealmQuestionsButton.heightAnchor.constraint(equalToConstant: 50)
+        ])
+    }
+    
+    
+    private func createRemoveButton() {
+        view.addSubview(removeFromRealmButton)
+        removeFromRealmButton.translatesAutoresizingMaskIntoConstraints = false
+        removeFromRealmButton.setTitle("Remove Question", for: .normal)
+        removeFromRealmButton.frame = view.frame
+        removeFromRealmButton.backgroundColor = .systemRed
+        removeFromRealmButton.layer.cornerRadius = 20
+        removeFromRealmButton.addTarget(self, action: #selector(removeButtonTapped), for: .touchUpInside)
+        
+        NSLayoutConstraint.activate([
+            removeFromRealmButton.topAnchor.constraint(equalTo: getRealmQuestionsButton.bottomAnchor, constant: 20),
+            removeFromRealmButton.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 20),
+            removeFromRealmButton.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -20),
+            removeFromRealmButton.heightAnchor.constraint(equalToConstant: 50)
+        ])
+    }
+    
+    
+    @objc func buttonTapped() {
+        let persistanceService = PersistanceService()
+        let questionsFromRealm = persistanceService.getQuestions()
+        
+        for question in questionsFromRealm {
+            print("Question: \(question.title) || Answer: \(question.correctAnswer)")
+        }
+    }
+    
+    
+    @objc func removeButtonTapped() {
+        let persistanceService = PersistanceService()
+        let questionsFromRealm = persistanceService.getQuestions()
+        
+        print("Questions from Realm before removal: \(questionsFromRealm[0])")
+        
+        let user = User()
```

Comment:
Maybe make this user global for the time being before we figure out how rammed it properly accessible. That way we can use it to test adding and removing various questions

Reply:
Looks like we already have a global one you can use

Reply:
I've remove this completely. This was based on the User object removing a Question object directly from Realm, which we've agreed isn't the best way to do this. So just deleting this code and going to implement the logic once I've updated the User object.